src/main/resources/templates/userinfo.mustache

```diff
@@ -161,6 +161,22 @@
                             {{return_code_claim}}
                         </dd>
                     </div>
+                  <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                      Wallet Subject ID Scope present
+                    </dt>
+                    <dd class="govuk-summary-list__value" id="user-info-wallet-subject-id-scope-present">
+                      {{wallet_subject_id_claim_present}}
+                    </dd>
+                  </div>
+                  <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                      Wallet subject ID Scope
+                    </dt>
+                    <dd class="govuk-summary-list__value" id="user-info-wallet-subject-id-scope">
+                      {{wallet_subject_id_claim}}
```

Comment:
There's a bit of inconsistency here between scope and claim - but happy to be told that's deliberate

Reply:
Sort of deliberate - it's a scope but is also classed as a OL claim (like email and phone number).

Because email and phone number are already part of the `PersonClaims` (that `UserInfo` extends) there are built-in methods to retrieve them. The `walletSubjectID` then gets set as a claim when it is calculated ([this PR, `UserInfoService`, line 161](https://github.com/govuk-one-login/authentication-api/pull/4003/files#diff-28d48eb177704d80a55a1fc52e3267f42bdcf2583243c737c51dfe635b8370b1))

Happy to change if you can think of a less confusing way of putting it 😅